### PR TITLE
AC Automated Test - LRAC-8340, LRAC-8341, LRAC-8342 | master

### PR DESCRIPTION
**Pull request by 91nilo**
https://issues.liferay.com/browse/LRAC-8340
https://issues.liferay.com/browse/LRAC-8933 (Refactor)
https://issues.liferay.com/browse/LRAC-8341
https://issues.liferay.com/browse/LRAC-8934 (Refactor)
https://issues.liferay.com/browse/LRAC-8342
https://issues.liferay.com/browse/LRAC-8957 (Refactor)

### CODE REVIEW

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
Hi @alexcandidos and @ViniLopes20, i made some improvements, as requested, but if you think there is anything else i could do to make the code better, just let me know.
Thank you for the feedbacks.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase [hunk 1]
```diff
@@ -117,25 +117,169 @@ definition {
...
+		}
+
+		task ("Assert that the right page is present on custom time filter") {
+			ACUtils.assertItemsName(itemList = "Custom Filter AC Page");
+		}
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8340 | Test Summary: Filter Page Table by last 180 days"
+	@priority = "3"
+	test PeriodFilter180DaysPageTable {
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase:179]: Hi @91nilo, I think you forgot to put this on


```
task ("Assert that the right page is present on custom time filter") {
	ACUtils.assertItemsName(itemList = "Last 180 days Page");
}
```

Comment (91nilo) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase:179]: Fixed, thanks!

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase [hunk 2]
```diff
@@ -117,25 +117,175 @@ definition {
...
+				endMonth = "August",
+				endYear = "2021",
+				startDay = "1",
+				startMonth = "July",
+				startYear = "2021");
+		}
+
+		task ("Assert that the right page is present on custom time filter") {
+			ACUtils.assertItemsName(itemList = "Custom Filter AC Page");
+		}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase:174]: Hi @91nilo, I think you can validate that the **AC Page** is not present in Custom Range

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @91nilo , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**